### ISSUE TITLE
Example chain_2d crashes when clicking

### ISSUE THREAD

**Issue by kainosaur** [Jondolf/avian#416]
```
thread 'main' panicked at crates\avian2d\../../src\position.rs:252:9:
the given sine and cosine produce an invalid rotation
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Encountered a panic in system `avian2d::dynamics::solver::xpbd::solve_constraint<avian2d::dynamics::solver::joints::revolute::RevoluteJoint, 2>`!
Encountered a panic in system `avian2d::schedule::run_substep_schedule`!
Encountered a panic in system `avian2d::schedule::run_physics_schedule`!
Encountered a panic in system `bevy_app::main_schedule::Main::run_main`!
```

**Comment by Khanoto** [comment]
![image](https://github.com/user-attachments/assets/d8eb8c68-a7aa-4b85-bab0-50eca991a0dc)

Check the angles you're using

**Comment by kainosaur** [comment]
I am using [this example](https://github.com/Jondolf/avian/blob/main/crates/avian2d/examples/chain_2d.rs) unmodified.

**Comment by Khanoto** [comment]
Since you said it come from clicking, I think it may come from the FollowMouse func

But the only thing I notice is that it may be reversed between the follower pos and cursor pos
![image](https://github.com/user-attachments/assets/3ec6c464-828e-4f09-9354-c5464862d1bb)

just a suggestion, I haven't tried

**Comment by Jondolf** [comment]
Fixed by #421 :)
